src/reduce.test.js

```diff
@@ -11,9 +11,47 @@ describe('reduce', () => {
     delete Array.prototype.reduce2;
   });
 
-  it('should ', () => {
+  it('should reduce an array to a single value by applying a callback', () => {
+    const arr = [1, 2, 3, 4, 5];
 
+    const sum = arr.reduce2((acc, curr) => acc + curr, 0);
```

Comment:
What is the function name "reduce2" ?) fix it, should be declared valid and relevant variables name as per code convention)